components/brave_rewards/resources/ui/components/listToken/index.tsx

```diff
@@ -16,6 +17,7 @@ export interface Props {
   color?: Type
   border?: 'first' | 'last' | 'default'
   testId?: string
+  link?: string
```

Comment:
Nit: Seems a bit vague as a name. Maybe `transactionHistoryLink` or something similar?

Reply:
Actually, I thought it would be weird if we brought in a name in the `Token` `interface` / `ListToken` `class` that we only use in context of `ads` `SummaryItem`s. WDYT?